src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1517,19 +1501,21 @@ private function getProcessForCode($code, $cwd = null, array $env = null, $input
         return $this->getProcess(array(self::$phpBin, '-r', $code), $cwd, $env, $input, $timeout);
     }
 
-    private function skipIfNotEnhancedSigchild($expectException = true)
+    private function skipIfSigchildEnabled($expectException = true)
     {
-        if (self::$sigchild) {
-            if (!$expectException) {
-                $this->markTestSkipped('PHP is compiled with --enable-sigchild.');
-            } elseif (self::$notEnhancedSigchild) {
-                if (method_exists($this, 'expectException')) {
-                    $this->expectException('Symfony\Component\Process\Exception\RuntimeException');
-                    $this->expectExceptionMessage('This PHP has been compiled with --enable-sigchild.');
-                } else {
-                    $this->setExpectedException('Symfony\Component\Process\Exception\RuntimeException', 'This PHP has been compiled with --enable-sigchild.');
-                }
+        if (!self::$sigchild) {
+            return;
+        }
+
+        if ($expectException) {
+            if (method_exists($this, 'expectException')) {
```

Comment:
Ok but... I'm not sure I'll be able to fix this alone... Could you show me what I need to do more exactly please?